lib/structures/User.js

```diff
@@ -53,6 +58,13 @@ class User extends Base {
         return this.avatar ? this._client._formatImage(Endpoints.USER_AVATAR(this.id, this.avatar)) : this.defaultAvatarURL;
     }
 
+    get bannerURL() {
+        if(this._missingClientError) {
```

Comment:
We can probably return null if there is no banner before checking if the client exists